pkg/constants/constants.go

```diff
@@ -164,6 +164,12 @@ const (
 
 	// OSMMeshConfig is the name of the OSM MeshConfig
 	OSMMeshConfig = "osm-mesh-config"
+
+	// OSWindows is the name for Windows operating system.
+	OSWindows string = "windows"
+
+	// OSLinux is the name for Linux operating system.
+	OSLinux string = "linux"
 )
```

Comment:
Since we aren't using a custom string type, for grouping purposes I suggest using a new `const()` definition similar to the protocols defined below.

Reply:
fixed now